client/apis/volumegroupsnapshot/v1alpha1/types.go

```diff
@@ -333,6 +333,12 @@ type VolumeGroupSnapshotContentStatus struct {
 	// +optional
 	VolumeGroupSnapshotHandle *string `json:"volumeGroupSnapshotHandle,omitempty" protobuf:"bytes,1,opt,name=volumeGroupSnapshotHandle"`
 
+	// VolumeSnapshotHandlePairList is a list of CSI "volume_id" and "snapshot_id"
+	// pair returned by the CSI driver to identify snapshots and their source volumes
+	// on the storage system.
+	// +optional
+	VolumeSnapshotHandlePairList []VolumeSnapshotHandlePair `json:"volumeSnapshotHandlePairList,omitempty" protobuf:"bytes,6,opt,name=volumeSnapshotHandlePairList"`
```

Comment:
Can you move this to line 368 after item 5?  It's confusing to see it jump from 1 to 6.

Reply:
I followed the order specified in the KEP, but having it after item 5 is definitely better